api/src/test/java/io/strimzi/test/StrimziExtension.java

```diff
@@ -606,9 +609,39 @@ protected void after() {
         return last;
     }
 
+    private void applyMultipleNamespacesWatcher(AnnotatedElement element) {
+        List<Namespace> namespaces = annotations(element, Namespace.class);
+        String defaultNamespace  = namespaces.get(0).value();
+        for (Namespace namespace: namespaces) {
+            if (namespace.value().matches(defaultNamespace)) {
+                continue;
+            }
+            Map<File, String> configYamlFiles = Arrays.stream(
+                    Objects.requireNonNull(new File(CO_INSTALL_DIR).listFiles((file, name) -> name.matches("[0-9]*-RoleBinding.*")))
+            ).sorted().collect(Collectors.toMap(file -> file, f -> getContent(f, node -> {
+
+                ArrayNode subjects = (ArrayNode) node.get("subjects");
+                ObjectNode subject = (ObjectNode) subjects.get(0);
+                subject.put("kind", "ServiceAccount")
+                        .put("name", "strimzi-cluster-operator")
+                        .put("namespace", defaultNamespace);
+
+            }), (x, y) -> x, LinkedHashMap::new));
```

Comment:
I think we need to stop parsing these resources into `ObjectNode` : It makes it harder to understand what's going on. We should instead parse them into the Fabric8 resource and manipulate them using that typefsafe API.